config/initializers/captchas.rb

```diff
@@ -2,9 +2,12 @@
 
 Recaptcha.configure do |config|
   # Do not verify recaptcha keys if it is not correctly configured
-  (config.skip_verify_env ||= []) << Rails.env if Rails.configuration.three_scale.recaptcha_private_key.blank?
+  (config.skip_verify_env ||= []) << Rails.env if Rails.configuration.three_scale.recaptcha_public_key.blank?
   config.site_key = Rails.configuration.three_scale.recaptcha_public_key
   config.secret_key = Rails.configuration.three_scale.recaptcha_private_key
+  config.enterprise = Rails.configuration.three_scale.recaptcha_enterprise_enabled
```

Comment:
Can we use public and secret key settings where secret key would be the api key for enterprise like presently the public key is one setting for both?

Then the presence of `project_id` setting will tell us that enterprise endpoint should be used. That way we add only one new setting instead of 3. And avoid possible mixed configuration that can confuse users and support.

Just thinking.